fakeserver/routes.go

```diff
@@ -218,6 +268,23 @@ func getV1AppVisitorConfig() (interface{}, error) {
 	}, nil
 }
 
+func getV2AppVisitorConfig() (interface{}, error) {
+	isplitRegistry, err := getV2SplitRegistry()
```

Comment:
this name hints that there's a type ISplitRegistry in my head, but whatevs. hard to express this easily and succinctly.